src/Inputs/ComboBox.tsx

```diff
@@ -0,0 +1,241 @@
+import React, {
+    useState,
+    useEffect,
+    useCallback,
+    Children,
+    isValidElement,
+    SyntheticEvent, 
+} from 'react';
+import styled, { withTheme, DefaultTheme } from 'styled-components';
+import {
+    flex,
+    scroll,
+    position,
+    transition,
+    clickable,
+    darken,
+} from '@Utils/Mixins';
+import { useTransition } from '@Utils/Hooks';
+import { LabelLayout, LabelLayoutProps,InputFragment } from '@Layouts';
+
+const NUM_VISIBLE_SELECTIONS = 3;
+
+var options:any = [];

```

Comment:
No >:c bad

Reply:
![](https://media.giphy.com/media/m6etefcEsTANa/giphy.gif)

Reply:
:)))
everything from 23 to 29?

Reply:
I'm using options array in a useMemo and changing window type from any to Window & typeof globalThis

Reply:
I'm using options in a useMemo hook now